static/app/styles/animations.tsx

```diff
@@ -124,3 +123,28 @@ export const alertHighlight = (priority: string, theme: Theme) => keyframes`
     border-color: ${theme.alert[priority].border};
   }
 `;
+
+export const shake = ({
+  steps = 50,
+  intensity = 3,
+}: {steps?: number; intensity?: number | [number, number]} = {}) => keyframes`
+${new Array(steps)
+  .fill(0)
+  .map((_, i) => {
+    const [intensityX, intensityY] = Array.isArray(intensity)
+      ? intensity
+      : [intensity, intensity];
+
+    const rand1 = Math.round(Math.random() * intensityX);
+    const rand2 = Math.round(Math.random() * intensityY);
```

Comment:
Should this animation be random? You could end up with subtle movement or much more in a non-deterministic way. If you had a fixed 'jiggle' amount you'd get more consistent results.

Reply:
Yeah you're right

Reply:
It might be tricky to get the easing right

Reply:
Ok just removed it from here and simplified a bunch